doc/api/permissions.md

```diff
@@ -586,7 +586,8 @@ There are constraints you need to know before using this system:
 * The Permission Model is initialized after the Node.js environment is set up.
   However, certain flags such as `--env-file` or `--openssl-config` are designed
   to read files before environment initialization. As a result, such flags are
-  not subject to the rules of the Permission Model.
+  not subject to the rules of the Permission Model. The same applies for V8
+  flags that can be set via runtime through `v8.setFlagsFromString`.
```

Comment:
```suggestion
  flags that can be set via runtime through methods such as `v8.setFlagsFromString`.
```

IMO using 'such as' allows accounts for any future functions with this behavior that might happen without updating this document.